### DISCUSSION TITLE
Need some help with "nested" wildcards or wildcards within wildcards.

### DISCUSSION THREAD

**Discussion by StanleyPain** [adieyal/sd-dynamic-prompts#412] [Q&A] [answered]
I'm trying to use the jumbo nationalities collection but it seems that when I call the wildcard like this: `__jumbo/people/nationalities/asia__` I only get `__people/nationalities/southeast-asia__` as the output.

What am I doing wrong here?

**Comment by StanleyPain** [comment]
There are supposed to be two underscores in front and behind that but I guess markdown did it's thing :S

**Comment by akx** [comment, marked as answer]
This is probably https://github.com/adieyal/sd-dynamic-prompts/issues/377 - I haven't had the time to fix this yet, so a workaround is to edit the references in the `jumbo/people/nationalities/asia.txt` file to be `jumbo/....` prefixed.

**Reply by StanleyPain** [reply]
I ended up doing just that. Thanks :)
